src/GrumPHP/Task/Phpunit.php

```diff
@@ -29,9 +29,11 @@ public function getConfigurableOptions()
         $resolver = new OptionsResolver();
         $resolver->setDefaults(array(
             'config_file' => null,
+            'group' => null,
         ));
 
         $resolver->addAllowedTypes('config_file', array('null', 'string'));
+        $resolver->addAllowedTypes('group', array('null', 'array'));
```

Comment:
Isn't it better to set the default value to an empty array and only allow array types?